src/include/catalog/catalog_defs.h

```diff
@@ -3,6 +3,8 @@
 #include "common/strong_typedef.h"
 
 namespace terrier::catalog {
+STRONG_TYPEDEF(offset_oid_t, uint32_t);
```

Comment:
Not sure you need an oid type for offsets. Postgres defines oid as essentially the primary key for `pg_class` I believe. So unless something can be looked up in `pg_class`, it probably shouldn't be an oid